hikari/api/rest.py

```diff
@@ -1121,7 +1360,42 @@ async def edit_webhook(
         channel: undefined.UndefinedOr[snowflakes.SnowflakeishOr[channels.TextChannel]] = undefined.UNDEFINED,
         reason: undefined.UndefinedOr[str] = undefined.UNDEFINED,
     ) -> webhooks.Webhook:
-        ...
+        """Edit a webhook.
+
+        Parameters
+        ----------
+        webhook : hikari.snowflakes.SnowflakeishOr[hikari.webhooks.Webhook]
+            The webhook to edit. This may be a `hikari.webhooks.Webhook`
+            or the ID of an existing webhook.
+        token : hikari.undefined.UndefinedOr[builtins.str]
+            If specified, the webhook token to use to edit the webhook.
```

Comment:
So... what happens if you don't give a token, we should explain the behavior around this a bit, perhaps.